backend/db/migrate/20230606204847_create_friendship.rb

```diff
@@ -0,0 +1,10 @@
+class CreateFriendship < ActiveRecord::Migration[7.0]
+  def change
+    create_table :friendships do |t|
```

Comment:
What do you think about this table having dups of everything?  ie having both directions of friendships saved, so the outgoing friendship direction query is simple